assets/js/modules/analytics/components/common/PropertySelect.test.js

```diff
@@ -111,9 +113,13 @@ describe( 'PropertySelect', () => {
 
 		await act( () => registry.dispatch( STORE_NAME ).setAccountID( ACCOUNT_CREATE ) );
 
-		// ACCOUNT_CREATE is an invalid (but valid selection), so ensure the select IS currently disabled.
-		expect( selectWrapper ).toHaveClass( 'mdc-select--disabled' );
-		expect( selectedText ).toHaveAttribute( 'aria-disabled', 'true' );
+		// ACCOUNT_CREATE is an invalid (but valid selection), so ensure the select is not rendered
```

Comment:
Similar to the previous comment, let's clarify this and add at least what seems to be a missing noun 🙂

Reply:
Updated.